x-pack/plugins/enterprise_search/public/applications/app_search/components/synonyms/synonyms_logic.test.ts

```diff
@@ -121,5 +152,163 @@ describe('SynonymsLogic', () => {
         expect(flashAPIErrors).toHaveBeenCalledWith('error');
       });
     });
+
+    describe('createSynonymSet', () => {
+      it('should set modalLoading state and clear flash messages', () => {
```

Comment:
Hm, personally from a dev/doc perspective I prefer to view all the side effects of an action in a single place rather than jumping up and down the file. So while listeners may have reducer effects, I think we could still keep it in its parent `describe('someListener')` block and give it a child `it('affects some value')` test and user the helper. WDYT?

Reply:
Yep, I think that makes sense. Thanks.